tests/unit/amo/components/TestSearch.js

```diff
@@ -153,6 +159,12 @@ describe('Search', () => {
     expect(root.find(ErrorList)).toHaveLength(1);
   });
 
+  it('should render "Enter a search term and try again." msg if no search term', () => {
```

Comment:
The spec doesn't need the exact error text; `it('should render an error message on empty search', () => {` would be better.